DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/vlan/test_vlan_priority.py

```diff
@@ -0,0 +1,98 @@
+import asyncio
+
+import pytest
+from dent_os_testbed.utils.test_utils.tgen_utils import (tgen_utils_get_dent_devices_with_tgen,
+                                                         tgen_utils_setup_streams, tgen_utils_start_traffic,
+                                                         tgen_utils_stop_traffic, tgen_utils_get_traffic_stats,
+                                                         tgen_utils_get_loss, tgen_utils_dev_groups_from_config,
+                                                         tgen_utils_traffic_generator_connect,
+                                                         tgen_utils_get_egress_stats)
+from dent_os_testbed.utils.test_utils.br_utils import (configure_bridge_setup, configure_vlan_setup,
+                                                       get_traffic_port_vlan_mapping)
+
+pytestmark = [pytest.mark.suite_functional_vlan,
+              pytest.mark.asyncio,
+              pytest.mark.usefixtures("cleanup_bridges", "cleanup_tgen")]
+
+
+async def test_vlan_priority_tag(testbed):
+    """
+    Test Name: VLAN priority
+    Test Suite: suite_functional_vlan
+    Test Overview: Test packet forwarding with VLAN priority
+    Test Procedure:
+    1. Initiate test params.
```

Comment:
Too general description. It's not clear what happens and what is expected.
I'm about all steps.